tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/pullquote_block.yaml

```diff
@@ -15,6 +15,6 @@ tags:
     avatar width-143 format-webp class="avatar__image":
       raw: |
         <img src="https://source.unsplash.com/143x143?face" width="143" height="143" alt="Some alt" class="avatar__image">
-    value.company_logo fill-110x70 format-webp class="blockquote__company-logo":
```

Comment:
this is where there might be some issues as this classname has since been updated